torch/csrc/jit/frontend/tracer.cpp

```diff
@@ -429,7 +430,9 @@ static void gatherParametersAndBuffers(
     if (isCustomClass(s.value)) {
       tracer::setValueTrace(s.value, trace_get_attr);
     }
-    if (self_ty->getAttribute(s.name)->is_module()) {
+    auto attr_type = self_ty->getAttribute(s.name);
```

Comment:
Can you leave a comment here explaining why we are skipping the interfacetype?

Reply:
Done.